shunting-yard-receiver/shunting-yard-receiver-sqs/pom.xml

```diff
@@ -2,9 +2,9 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>com.hotels</groupId>
+    <groupId>com.expediagroup</groupId>
     <artifactId>shunting-yard-receiver</artifactId>
-    <version>2.0.2-SNAPSHOT</version>
+    <version>3.0.0-SNAPSHOT</version>
   </parent>
 
   <artifactId>shunting-yard-receiver-sqs</artifactId>
```

Comment:
Dependency on `shunting-yard-common` below needs the group id updated.

Reply:
done.

Reply:
done.